samples/uikit/konan/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -39,6 +39,11 @@
       "idiom" : "iphone",
       "size" : "60x60",
       "scale" : "3x"
+    },
+    {
+      "idiom" : "ios-marketing",
```

Comment:
Hmm, what is this?

Reply:
Some artifacts added by Xcode while editing the project.